int8 throttle_little
int8 throttle_big
char status
